server/pages/index.pug

```diff
@@ -226,6 +226,56 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
+		script.
+			(function () {
+				// Mini tracks
+				function recordTracksEvent( eventName ) {
+					var statsPixel = 'http://pixel.wp.com/t.gif';
+					var stats = {
+						_pf: navigator.platform,
+						_lg: navigator.language,
+						_en: eventName,
+						_ht: window.screen.height,
+						_wd: window.screen.width,
+						_ut: 'anon',
+						_dl: document.location.origin + '/' + document.location.pathname,
+						_ts: Date.now(),
+						_tz: new Date().getTimezoneOffset() / 60
```

Comment:
tracks appends a `rand` param too with the results of `Math.random()` to cache-bust queries. _ts is sort of doing the same thing, but it might be better to replicate what the real Tracks client does